src/rpa_sigma_functional.F

```diff
@@ -731,15 +731,15 @@ INTEGER FUNCTION intervalnum(x_coord, n, sigma) RESULT(inum)
 
       INTEGER                                            :: i
 
-      IF (sigma <= 0.0_dp) STOP 'intervalnum: sigma should be positive'
+      IF (sigma <= 0.0_dp) CPABORT('intervalnum: sigma should be positive')
 
       inum = -1
       IF (sigma > x_coord(n)) inum = n
       DO i = 1, n - 1
          IF ((sigma > x_coord(i)) .AND. (sigma <= x_coord(i + 1))) inum = i
       END DO
 
-      IF (inum == -1) STOP 'interval: something was wrong'
+      IF (inum == -1) CPABORT('interval: something was wrong')
```

Comment:
One more comment, I changed the STOP in this file to CPABORT, in order to satisfy the conventions checker. I assume this is fine with @Raviraj26 (this was introduced in #3695 )?